### PR TITLE
added documentation for getFileContent()

### PR DESCRIPTION
**Pull request by dghelm**
First commit to the Slate docs, adding documentation for `getFileContent()` since it could be important for some upcoming partnerships.

First PR for the Slate docs, so review carefully :)

### CODE REVIEW

File: source/v3-beta/includes/_browser-js-api.md [hunk 1]
```diff
@@ -30,6 +30,57 @@ See [Downloading A File](#downloading-a-file).
...
+    length: 65,
+    subfiles: { 'example_1.json': [Object] }
+  },
+  skylink: 'sia:AACJjVpOpsZ6c9PBwOYvxTtDc_nmrGxTTEomHDBEEfvRhA'
+}
+```
+
+Field | Description
+------| -----------
+`data` | Data returned in the request body when doing a GET request for the skylink.
```
Comment (m-cat) [inline source/v3-beta/includes/_browser-js-api.md:75]:
```suggestion
`data` | Data returned in the response body when doing a GET request for the skylink.
```

File: source/v3-beta/includes/_browser-js-api.md [hunk 2]
```diff
@@ -30,6 +30,57 @@ See [Downloading A File](#downloading-a-file).
...
+
+```javascript--browser
+import { SkynetClient } from "skynet-js";
+
+const client = new SkynetClient();
+const skylink = "AACJjVpOpsZ6c9PBwOYvxTtDc_nmrGxTTEomHDBEEfvRhA"; // JSON File
+let skyfile = {}
+
+try {
+  skyfile = await client.getFileContent(skylink);
```
Comment (m-cat) [inline source/v3-beta/includes/_browser-js-api.md:43]: It is more convenient to get the data like this so maybe we should showcase it:

```suggestion
  const { data } = await client.getFileContent(skylink);
```

**Review by m-cat — CHANGES_REQUESTED** [review]
Looks mostly good, thanks!

**Comment by dhelma** [comment]
Should be good to go.

**Review by m-cat — APPROVED** [review]

**Pull request merged**